### PR TITLE
fix: add border and border radius property for label tag

### PR DESCRIPTION
**Pull request by itssharmasandeep**

### CODE REVIEW

File: projects/components/src/label-tag/label-tag.component.ts [hunk 1]
```diff
@@ -30,4 +38,14 @@ export class LabelTagComponent {
...
   public prefixIcon?: string;
+
+  @Input()
+  public borderColor?: Color;
+
+  @Input()
+  public borderRadius: number = 0; // In Pixels
+
+  public get border(): string | undefined {
+    return this.borderColor === undefined ? undefined : `1px solid ${this.borderColor}`;
```
Comment (adisreyaj) [inline projects/components/src/label-tag/label-tag.component.ts:49]: Why not have the border (transparent) always so that there is no layout change.

Comment (itssharmasandeep) [reply projects/components/src/label-tag/label-tag.component.ts:49]: I don't think anyone would be setting and un-setting the borderColor for this particular component and didn't want to use border by default (just a preference). If needed, I can make this change

Comment (anandtiwary) [reply projects/components/src/label-tag/label-tag.component.ts:49]: We should ideally have a  `viewMode` enum property. The caller would pass a view property, and we would apply the style from inside this component. We need to restrict this component honestly. By adding new CSS properties, we are making it too generic.

Comment (arjunlalb) [reply projects/components/src/label-tag/label-tag.component.ts:49]: +1 on the viewmode enum.

Comment (itssharmasandeep) [reply projects/components/src/label-tag/label-tag.component.ts:49]: But in that case, how would we have the border-specific configurations?
Then will probably take text color as border color may be?

**Comment by itssharmasandeep** [comment]
Not needed

**Pull request closed without merging**